src/network/graphDataStruct.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'lodash';
```

Comment:
wanna any lib handling import schema.js from grapqhql, so dunt need re-define here

Reply:
Ya I was wondering about that too. Will look into it.